src/Services/QueueMonitor.php

```diff
@@ -174,6 +218,17 @@ protected static function jobFinished(JobContract $job, int $status, ?\Throwable
             return;
         }
 
+        // if the job has an exception, but it's not failed (it did not exceed max tries and max exceptions),
```

Comment:
@romanzipp please take a look here
should we consider these cases as queued when the config option is disabled?

actually, Laravel returns these cases to the queue but I mean when the config option is disabled